src/Microsoft.Health.Dicom.Tests.Common/AcceptHeaderHelpers.cs

```diff
@@ -56,6 +56,15 @@ public static AcceptHeader CreateRenderJpegAcceptHeader(string transferSyntax =
           quality: quality);
     }
 
+    public static AcceptHeader CreateRenderPngAcceptHeader(string transferSyntax = "*", string mediaType = KnownContentTypes.ImagePng, double? quality = null, PayloadTypes payloadType = PayloadTypes.SinglePart)
```

Comment:
why quality is double here?

Reply:
this quality is used for something else in the accept header helpers - not for render

Reply:
do we need this?

Reply:
update